packages/framework/src/ShopsysFrameworkBundle.php

```diff
@@ -13,6 +13,11 @@
 
 class ShopsysFrameworkBundle extends Bundle
 {
+    /**
+     * @var string
+     */
+    public const VERSION = '7.2.0-dev';
```

Comment:
The really cool would be to have link to documentation here. 

UpdateFrameworkKernelVersionReleaseWorker would update it to something like `https://github.com/shopsys/shopsys/blob/v7.2.0/docs/index.md` and `SetDevFrameworkKernelVersionReleaseWorker` back to `https://github.com/shopsys/shopsys/blob/master/docs/index.md`

Reply:
Hi, I solved it differently to not need to update another thing in release worker and it's bounded to the version constant now.

Reply:
Your solution is much better than mine, good job!